Makefile

```diff
@@ -0,0 +1,24 @@
+BINARY=argocd-vault-plugin
+VERSION=0.1
+OS_ARCH=darwin_amd64
+
+default: build
+
+format: 
+	go fmt github.com/IBM/argocd-vault-plugin/...
+
+quality:
+	go vet
```

Comment:
I think you need to do `go vet ./...`

Reply:
Yeah I had a suspicion but it didn't seem to do anything either way, thanks for telling me